cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -84,149 +113,234 @@ public void handlerReturnsAnNvaPublicationEntryWhenInputIsEventWithCristinResult
     }
 
     @Test
-    public void handlerSavesPublicationToDynamoDbWhenInputIsEventWithCristinResult() throws JsonProcessingException {
-        String input = validEventToJsonString();
-        handler.handleRequest(stringToStream(input), outputStream, CONTEXT);
-        CristinObject cristinObject = generatePublicationFromResource(input);
-        Publication expectedPublication = cristinObject.toPublication();
-        UserInstance userInstance = extractUserInstance(expectedPublication);
+    public void handlerSavesPublicationToDynamoDbWhenInputIsEventWithCristinResult() {
+        CristinObject cristinObject = cristinDataGenerator.randomObject();
+        AwsEventBridgeEvent<FileContentsEvent<JsonNode>> awsEvent = cristinDataGenerator.toAwsEvent(cristinObject);
+        InputStream input = stringToStream(awsEvent.toJsonString());
```

Comment:
These 3 lines are repeated throughout the code and they are not a method because we need all 3 values in most cases.